template/main.template.dart

```diff
@@ -0,0 +1,6 @@
+/* installed import
```

Comment:
Whats the difference between a installed and a template import? 🤔(Omg i just see how this smiley i love to use looks like on windows)